src/js/components/Grid/stories/AreasPropAlternative.js

```diff
@@ -1,37 +1,37 @@
 import React from 'react';
 
-import { Grommet, Box, Grid } from 'grommet';
-import { grommet } from 'grommet/themes';
+import { Box, Grid } from 'grommet';
 
 export const GridAreasAlternative = () => (
-  <Grommet full theme={grommet}>
-    <Grid
-      rows={['xxsmall', 'medium', 'xsmall']}
-      columns={['1/4', '3/4']}
-      areas={[
-        ['header', 'header'],
-        ['sidebar', 'main'],
-        ['footer', 'footer'],
-      ]}
-      gap="small"
-    >
-      <Box background="brand" gridArea="header">
-        Header
-      </Box>
+  // Uncomment <Grommet> lines when using outside of storybook
+  // <Grommet theme={...}>
+  <Grid
+    rows={['xxsmall', 'medium', 'xsmall']}
+    columns={['1/4', '3/4']}
+    areas={[
+      ['header', 'header'],
+      ['sidebar', 'main'],
+      ['footer', 'footer'],
+    ]}
+    gap="small"
+  >
+    <Box background="brand" gridArea="header">
+      Header
+    </Box>
 
-      <Box background="light-5" gridArea="sidebar">
-        Sidebar
-      </Box>
+    <Box background="light-5" gridArea="sidebar">
+      Sidebar
+    </Box>
 
-      <Box background="light-2" gridArea="main">
-        Main
-      </Box>
+    <Box background="light-2" gridArea="main">
+      Main
+    </Box>
 
-      <Box background="dark-2" gridArea="footer">
-        Footer
-      </Box>
-    </Grid>
-  </Grommet>
+    <Box background="dark-2" gridArea="footer">
+      Footer
+    </Box>
+  </Grid>
+  // </Grommet>
 );
 
```

Comment:
```suggestion

GridAreasAlternative.args = {
  full: true,
};

```
We should add the full prop to this story